src/axom/quest/MarchingCubes.cpp

```diff
@@ -217,7 +239,7 @@ void MarchingCubesSingleDomain::allocateImpl()
       : std::unique_ptr<ImplBase>(
           new MarchingCubesImpl<3, axom::SEQ_EXEC, axom::SEQ_EXEC>);
   }
-#ifdef _AXOM_MC_USE_OPENMP
+  #ifdef _AXOM_MC_USE_OPENMP
```

Comment:
A couple of comments/question:
1. I think the macro name is confusing since "mc" more commonly refers to Monte Carlo. Change "MC" to something like "MARCHCUBES"?
2. Shouldn't the goal be to make the execution selectable at runtime (e.g., user specifies programming model choice via API call) rather than a configuration choice?

Reply:
Changing AXOM_MC to AXOM_MARCHINGCUBES.
We do support run-time model selection.  These macros remove parallel modes that axom wasn't configured with.